src/pages/LandingPage_.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import '../styles/pages/LandingPage_.scss';
```

Comment:
Same feedback. Do not use underscore with file names. For the components name it should always be Pascal case and any other utilities/services/modules use camelCase